apps/i18n/javalab/en_us.json

```diff
@@ -19,8 +19,9 @@
   "closeReview": "Close review",
   "codeEditingDisabled": "Code editing is disabled while your project is in review",
   "codeReviewForStudent": "Code Review for {student}",
-  "codeReviewForYou": "Your Code Review",
   "codeReviewDisabledMessage": "Code review is disabled. Your teacher must place you in a code review group to enable this feature.",
+  "codeReviewForYou": "Your Code Review",
+  "codeReviewingPeer": "Code Reviewing {peerName}",
```

Comment:
Not sure if there's a change needed, but this is now very similar to `codeReviewForStudent` above.